src/index.js

```diff
@@ -1,57 +1,106 @@
-
-    ul.appendChild(li);
-    ul.appendChild(hr);
-  });
-};
-displayLi();
+import ToDoList from './modules/ToDoList.js';
+import UI from './modules/UI.js';
+
+class Store {
+  static getToDoList() {
+    let todoList;
+    if (localStorage.getItem('todoList') === null) {
+      todoList = [];
+    } else {
+      todoList = JSON.parse(localStorage.getItem('todoList'));
+    }
+    return todoList;
+  }
+
+  static displaytoDolist() {
+    const todo = Store.getToDoList();
+    todo.forEach((todoItem, index) => {
+      const ui = new UI();
+      ui.displayToDo(todoItem, index);
+    });
+  }
 
-const updateCheck = (e) => {
-  if (e.target.checked === true) {
-    e.target.nextElementSibling.classList.add('completed');
+  static addList(todoItem) {
+    const todoList = Store.getToDoList();
+    todoList.push(todoItem);
+    localStorage.setItem('todoList', JSON.stringify(todoList));
   }
 
-  if (!e.target.checked) {
-    e.target.nextElementSibling.classList.remove('completed');
+  static removeList(target) {
+    const li = target.parentElement.parentElement;
+    const hr = li.nextElementSibling;
+    const itemRemoved = parseInt(li.querySelector('label').htmlFor, 10);
+    if (!itemRemoved) {
+      return;
+    }
+    const todoList = Store.getToDoList();
+    const findIndex = todoList.findIndex((index) => index === itemRemoved);
+    todoList.splice(findIndex, 1);
+    localStorage.setItem('todoList', JSON.stringify(todoList));
+    hr.remove();
+    li.remove();
   }
-};
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  const ui = new UI();
+  const inputList = document.getElementById('myInput');
+  inputList.addEventListener('keypress', (event) => {
+    const list = inputList.value;
+    if (!list) {
+      return null;
+    }
+
+    if (event.key === 'Enter') {
+      const newToDo = new ToDoList(list);
+      ui.displayToDo(newToDo, Store.getToDoList().length);
+      Store.addList(newToDo);
+      ui.clearFieldInput();
+      ui.registerEventListeners();
+      event.preventDefault();
+    }
+    return null;
+  });
+
+  Store.displaytoDolist();
+});
+
+document.addEventListener('DOMContentLoaded', () => {
+  const ui = new UI();
+  const inputList = document.getElementById('myInput');
+  inputList.addEventListener('keypress', (event) => {
+    const list = inputList.value;
+    if (!list) {
+      return null;
+    }
 
-const checkboxes = document.querySelectorAll('.checkbox');
-checkboxes.forEach((checkbox) => {
-  checkbox.addEventListener('click', updateCheck);
+    if (event.key === 'Enter') {
+      const newToDo = new ToDoList(list);
+      ui.displayToDo(newToDo, Store.getToDoList().length);
+      Store.addList(newToDo);
+      ui.clearFieldInput();
+      ui.registerEventListeners();
+      event.preventDefault();
+    }
+    return null;
+  });
+
+  Store.displaytoDolist();
 });
+
+document.getElementById('items').addEventListener('click', (e) => {
+  if (e.target.classList.contains('checkbox')) {
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20editing%20task%20descriptions.), Please Implement a function for editing task descriptions.